### ISSUE TITLE
Flaky ostree simplified installer test stuck on CS8

### ISSUE THREAD

**Issue by henrywang** [osbuild/osbuild-composer#2413]
To continue test, same VM has to be destroyed.

```
[centos@centos-stream-8 ~]$ sudo virsh list --all
 Id   Name                                             State
----------------------------------------------------------------
 3    edge-7da686c7-cff3-46cb-98d6-c0814a26ad54-uefi   running

[centos@centos-stream-8 ~]$ sudo virsh destroy edge-7da686c7-cff3-46cb-98d6-c0814a26ad54-uefi
Domain 'edge-7da686c7-cff3-46cb-98d6-c0814a26ad54-uefi' destroyed

[centos@centos-stream-8 ~]$ sudo virsh list --all
 Id   Name                                             State
----------------------------------------------------------------
 4    edge-7da686c7-cff3-46cb-98d6-c0814a26ad54-uefi   running
```

**Comment by jrusz** [comment]
This now also happened on CS-9 and RHEL-9 as well... 
RHEL-9: https://gitlab.com/osbuild/ci/osbuild-composer/-/jobs/2249057799 on compose RHEL-9.0.0-20220324.2
and CS-9: https://gitlab.com/osbuild/ci/osbuild-composer/-/jobs/2246004901 (it has been manually unstuck here the same way like cs-8)

**Comment by ondrejbudai** [comment]
Happened a lot during last week: https://www.osbuild.org/ci-flakiness/2022-03-28T08:57:39Z.html

**Comment by jrusz** [comment]
Yeah it's now happening also on RHEL-9.0 and 8.6 nightly and I missed one new `virt-install` in the test, adding workaround here: https://github.com/osbuild/osbuild-composer/pull/2487

**Comment by henrywang** [comment]
It's fixed by PR #2487
